Makefile

```
@@ -55,6 +55,41 @@ image:
 seed-demo-data:
 	bash tools/seed-demo-data.sh
 
+# ---------------------------------------------------------------------------
+# Local Kubernetes development (kind + Skaffold)
+# See docs/dev/local-kubernetes.md
+# ---------------------------------------------------------------------------
+KIND_CLUSTER ?= gomodel-dev
+
+# Create the local kind cluster and deploy in-cluster dependencies (idempotent).
+kind-up:
```

checkmake (0.3.2)

[warning] Target body for "kind-up" exceeds allowed length of 5 lines (9).

(maxbodylength)